etl/steps/grapher/dummy/2020-01-01/dummy.py

```diff
@@ -1,23 +0,0 @@
-from owid import catalog
```

Comment:
Those dummy steps are now committed in our codebase, you don't have to delete them (e.g. after trying out walkthrough).